ZeldaFullEditor/AreaSpecificBGColor.asm

```diff
@@ -71,10 +71,12 @@ IntColorLoad2:
 
     SEP #$30 ; Set A in 8bit mode
 
-    LDA $8140 : BEQ .custom ; pc 140140 is where ZS saves whether to use the asm or not
+    LDA $8140 : BNE .custom ; pc 140140 is where ZS saves whether to use the asm or not
 
     REP #$30 ; Set A in 16bit mode
 
+    LDX.w #$2A32
```

Comment:
What's being indexed here in the upcoming jump?

Reply:
oh it was a default color but its a good thing you pointed this out because i just realized this is the wrong asm and i have another file i was working in that is the real actual working .asm